### ISSUE TITLE
Available for New Android versions?

### ISSUE THREAD

**Issue by natrius** [SubhrajyotiSen/oandbackup#1]
Hello, this seems to be the latest fork of oandbackup. Will you make the APP available for 7.1.2 vor 8.0 ?
Do you try to take care about the supported Bugs reported at Jens Main Branch?

And, will there be an .apk available AS this APP can not be found on F-droid

Greets

**Comment by SubhrajyotiSen** [comment]
Hey! I am sorry I just read this issue. I haven't gotten much time recently but I'll definitely restart it ASAP and try my best to fix the bugs from the main project

**Comment by natrius** [comment]
Don't worry, good to hear you want to continue. I would suggest - there are some pull requests at the main, not merged - you could als write them if they want to commit them here at your version. Maybe some things are solved with them already.

**Comment by SubhrajyotiSen** [comment]
Yes, I will do that. I accidentally nuked the commit history in my repo. So will fix that first and merge the needed pull requests. I just wish I had a bit more time to work on this

**Comment by natrius** [comment]
Hm, as i was just looking through, i saw today that jens is active again in the original branch. Maybe you can contribute there. So you don't need to do all the work by yourself :)
So its not everything on your shoulders.
